corehq/apps/domain/utils.py

```diff
@@ -124,3 +124,22 @@ def get_serializable_wire_invoice_general_credit(general_credit):
         }]
 
     return []
+
+
+@quickcache(['domain_name'])
+def get_allowed_ucr_expressions(domain_name):
```

Comment:
does it make sense to add `get_allowed_ucr_expressions` and `save_allowed_ucr_expressions` as `classmethod`s to `AllowedUCRExpressionSettings` since it's very specific to that model?